README.md

```diff
@@ -3,33 +3,76 @@
 __URLs__
 
 * Name all of the parts of the url that you can remember.  In your own words describe what they do.
+```
+Protocol: the connection in which you access the webpage?
+The port: typically 80. dont remember much else
+The path: the file path that of the website.
+the domain: the URL which represents an IP address
+the query: What is being searched for
+anchor tag: link to specific part of page
+
+```
 * Name the pieces of the following urls:
 	* `https://www.google.com/`
+	```
+	Protocol - domain -port
+	```
 	* `https://workbook.galvanize.com/cohorts/41/learning_experiences/367`
+	```
+protocol - domain - port - path
+	```
 	* `http://locahost:5000/animals/puppies?onlycute=1&size=medium#firstpuppy`
+	```
+protocol - domain - port - path - query - anchor?
+	```
 	* `https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#4xx_Client_Error`
+	```
+	protocol - domain - port - path - anchor?
+	```
 * Can a server use more than 1 port?
+```
+no
```

Comment:
I think this is a poorly written question. I think whoever wrote it is looking for `no` as an answer but in reality, yes a server can have more than 1 port.